app/templates/editor/level/level-edit-view.jade

```diff
@@ -66,7 +73,7 @@ block header
             a
               span.glyphicon-floppy-disk.glyphicon
 
-        if level.isType('ladder')
+        if level.isType('hero-ladder', 'course-ladder')
```

Comment:
Nitpick to use `isLadder` method, or do we need the extra specificity?

Reply:
We don't want to use `isLadder` here, because a straight `'ladder'` type level will actually not offer the choice of opponent team from level editor (since there is only one team). I actually realized that this had been broken the whole time, since the hero and course ladder levels _should_ have offered this choice, but didn't, since the type check was wrong.